publiccode.yml

```
@@ -0,0 +1,89 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: 'itowns'
+url: 'https://github.com/iTowns/itowns'
+landingURL: 'https://github.com/iTowns/itowns.git'
+softwareVersion: "2.44.2" # needs to be updated
+releaseDate: '2024-09-17'# needs to be updated
+softwareType: library
+logo: "https://github.com/iTowns/itowns/blob/master/examples/images/itowns_logo.png"
+inputTypes: # ToDo : complete
+  - application/json
+platforms:
+  - web
+categories:
+  - website-builder
+  - data-visualization
+  - it-development
+usedBy: # ToDo : complete and ask for permissions
+  - IGN (https://www.ign.fr/)
+  - Ciril Grp (https://www.ciril.net/fr/)
+  - Sogelink (https://www.sogelink.com/)
+  - Laboratoire Liris (https://liris.cnrs.fr/)
+  - Laboratoire Lastig (https://www.umr-lastig.fr/)
+roadmap: "https://www.itowns-project.org/"
```

Maybe a github project link? And update it with our common roadmap?

```suggestion
roadmap: "https://github.com/iTowns/itowns/projects"
```

---

The pb is that the roadmap in githubs doesn't seems to be updated

---

I prefer to keep 'https://www.itowns-project.org/' as it is more general and understandable (and also pretty much up to date..)